### PR TITLE
feat: add pm.execution.skipRequest support

### PR DESCRIPTION
**Pull request by vedkribhu**

### CODE REVIEW

File: test/integration/request-flow/stop-execution-from-sandbox.test.js [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+            }
+        ]);
+    });
+
+    it('should send request with correct URL, variables after stopExecution should not be reflected', function () {
+        var url1 = testRun.request.getCall(0).args[3].url.toString(),
+            expectedToken = 'modified-1:coll-value-2:env-value-3:modified-4';
+
+        expect(url1).to.equal('https://postman-echo.com/get?param=' + expectedToken);
+    });
```
Comment (coditva) [inline test/integration/request-flow/stop-execution-from-sandbox.test.js:290]: Let's test other apis that can cause side effects:
- `.setNextRequest`
- `.sendRequest`
- request auths which log to console.

Comment (vedkribhu) [reply test/integration/request-flow/stop-execution-from-sandbox.test.js:257]: Added for setNextRequest and sendRequest

File: lib/runner/extensions/item.command.js [hunk 2]
```diff
@@ -163,6 +177,12 @@ module.exports = {
                         });
                     }
 
+                    if (shouldSkipExecution(prereqExecutions)) {
+                        return callback && callback.call(this, null, {
```
[the remark this answers is not in the document]
Comment (vedkribhu) [reply lib/runner/extensions/item.command.js:183]: Added.

**Comment by vedkribhu** [comment]
Closing as we are changing the approach to: https://github.com/postmanlabs/postman-runtime/pull/1354/

**Pull request closed without merging**